# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-header/genhd.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

device.h caues major pain for the user-space, so do not include it ;-)

  - Rene Rebe <rene@exactcode.de>

--- ./include/linux/genhd.h.orig	2004-10-14 16:07:26.910854072 +0200
+++ ./include/linux/genhd.h	2004-10-14 16:07:04.280294440 +0200
@@ -12,7 +12,7 @@
 #include <linux/config.h>
 #include <linux/types.h>
 #include <linux/major.h>
-#include <linux/device.h>
+/*#include <linux/device.h>*/
 #include <linux/smp.h>
 #include <linux/string.h>
 #include <linux/fs.h>
